client-ui/src/app/components/lineage/lineage-highlights-toolbar/lineage-highlights-toolbar.component.less

```diff
@@ -61,4 +61,13 @@
     color: #ccc;
   }
 
+  .old-spline-warn {
+    margin-top: 15px;
```

Comment:
All definitions of margins & paddings should be devided by rem (which is 16px) in that box. It means, it can be 4px, 8px, 12px, 16px, 24px etc. wchi is 0.25rem, 0.5rem, 0.75rem, 1rem, 1.5rem,